content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -31,7 +31,7 @@ request used a deprecated API version.
 
 Example: `k8s.io/removed-release: "1.22"`
 
-Value **must** be in the format "<major>.<minor>". It is set to target the removal release
+Value **must** be in the format "\<MAJOR>\.\<MINOR>\". It is set to target the removal release
```

Comment:
Why this change?

Reply:
When i tested the page . there was blank spaces instead of "<major>.<minor>" that's why